app/messages/process_update_message.rb

```diff
@@ -62,6 +78,19 @@ def health_check_endpoint
       HashUtils.dig(health_check, :data, :endpoint)
     end
 
+    def readiness_health_check_type
+      HashUtils.dig(readiness_health_check, :type)
+    end
+
+    def readiness_health_check_invocation_timeout
+      HashUtils.dig(readiness_health_check, :data, :invocation_timeout)
+    end
+
+    def readiness_health_check_endpoint
+      HashUtils.dig(readiness_health_check, :data, :endpoint)
+    end
+
+    # TODO: prob add something here eventually
```

Comment:
🤔

Reply:
Haha this is why I didn't want to merge it into main right away. I didn't copy what healthcheck does here, because the tests don't require it. But I suspect at some point in our integration we will need to change it. And if we never change it, then I suspect we missed something in our feature. Hence the todo. :D

Reply:
Update: This TODO will be removed in the next story :)